leapp/cli/__main__.py

```diff
@@ -1,4 +1,3 @@
-import leapp.utils.i18n
```

Comment:
Are you sure this import should be removed? Sure it's unused in this file, but looking at the [PR](https://github.com/oamg/leapp/pull/364/files#diff-23d9ccc8e4f98ba7963856040ff788b2) where this line was introduced, it was unused even back then, so maybe this is intentional.

Reply:
Indeed this is intentional to setup the builtins overrides for _ and P_

Reply:
Good catch, thanks. Was on a killing spree, will fix

Reply:
As for setup.cfg we should keep [tool:pytest] section if we intend to keep flake8 check, otherwise
`pytest --cache-clear --flake8 -m flake8 leapp tests/scripts/*.py` will produce
```=================================== FAILURES ===================================
_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/leapp/cli/report/__init__.py:11:1: F401 'leapp.utils.clicmd.command_arg' imported but unused

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/leapp/dialogs/__init__.py:1:1: F401 'leapp.dialogs.components.NumberComponent' imported but unused
/home/ivasilev/projects/leapp/leapp/dialogs/__init__.py:1:1: F401 'leapp.dialogs.components.BooleanComponent' imported but unused
/home/ivasilev/projects/leapp/leapp/dialogs/__init__.py:1:1: F401 'leapp.dialogs.components.ChoiceComponent' imported but unused
/home/ivasilev/projects/leapp/leapp/dialogs/__init__.py:1:1: F401 'leapp.dialogs.components.MultipleChoiceComponent' imported but unused
/home/ivasilev/projects/leapp/leapp/dialogs/__init__.py:4:1: F401 'leapp.dialogs.renderer.CommandlineRenderer' imported but unused

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/leapp/libraries/stdlib/__init__.py:9:1: F401 'subprocess' imported but unused
/home/ivasilev/projects/leapp/leapp/libraries/stdlib/__init__.py:12:1: F401 'six' imported but unused

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/leapp/messaging/__init__.py:8:1: F401 'six.moves.configparser' imported but unused

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/leapp/reporting/__init__.py:1:1: F401 'json' imported but unused

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_messaging.py:29:29: F811 redefinition of unused 'repository_dir' from line 8
/home/ivasilev/projects/leapp/tests/scripts/test_messaging.py:40:18: F811 redefinition of unused 'repository_dir' from line 8
/home/ivasilev/projects/leapp/tests/scripts/test_messaging.py:66:23: F811 redefinition of unused 'repository_dir' from line 8
/home/ivasilev/projects/leapp/tests/scripts/test_messaging.py:75:26: F811 redefinition of unused 'repository_dir' from line 8

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_repository_actor_definition.py:23:27: F811 redefinition of unused 'repository_dir' from line 6

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_repository.py:33:16: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_repository.py:81:15: F811 redefinition of unused 'repository_dir' from line 7

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:16:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:45:18: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:52:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:62:47: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:75:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:100:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:125:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:152:23: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:173:23: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:199:20: F811 redefinition of unused 'repository_dir' from line 7
/home/ivasilev/projects/leapp/tests/scripts/test_snactor.py:215:25: F811 redefinition of unused 'repository_dir' from line 7

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_stdlib_call.py:86:1: F811 redefinition of unused 'test_callability_check' from line 80

_________________________________ FLAKE8-check _________________________________
/home/ivasilev/projects/leapp/tests/scripts/test_utils_project.py:21:30: F811 redefinition of unused 'repository_dir' from line 6
/home/ivasilev/projects/leapp/tests/scripts/test_utils_project.py:58:34: F811 redefinition of unused 'repository_dir' from line 6
/home/ivasilev/projects/leapp/tests/scripts/test_utils_project.py:65:34: F811 redefinition of unused 'repository_dir' from line 6
```